test/fixtures/with-telemetry/input/app/templates/components/file-less.hbs

```diff
@@ -1,2 +1,2 @@
 <div>this template has no js </div>
-{{#bs-button type="primary"}}>Primary{{/bs-button}}
```

Comment:
I feel like we shouldn't be changing this on the input 🤔

Reply:
yes but I think that was originally committed to master by mistake, because of the situation that integration tests are run in-situ

Reply:
🎉